python/ray/_private/runtime_env/pip.py

```diff
@@ -329,6 +329,28 @@ async def _create_or_get_virtualenv(
             )
         await check_output_cmd(create_venv_cmd, logger=logger, cwd=cwd, env=env)
 
+    @classmethod
+    async def _install_uv(path: str, cwd: str, pip_env: Dict, logger: logging.Logger):
+        """`uv` shows much better performance than `pip` to install packages, so
+        leverage `uv` to for runtime env setup.
+        Before package install, make sure `uv` is installed.
+        """
+        virtualenv_path = _PathHelper.get_virtualenv_path(path)
+        python = _PathHelper.get_virtualenv_python(path)
+
+        uv_install_cmd = [
+            python,
+            "-m",
+            "pip",
+            "install",
+            "--disable-pip-version-check",
+            "--no-cache-dir",
+            "uv",
```

Comment:
Perhaps it would be useful if the user could specify the desired `uv` version? 
While `uv` is pretty stable unexpected things can always happen. 
Pinning all dependencies is a good practice for production.

Reply:
Would [it](https://github.com/ray-project/ray/pull/48457) make sense to you?

Reply:
Yep (left a small nit in the other PR)